src/Dfe.PlanTech.Web/Views/Shared/NotFoundError.cshtml

```diff
@@ -0,0 +1,14 @@
+<div class="govuk-grid-row">
+	<div class="govuk-grid-column-two-thirds">
+		<h1 class="govuk-heading-xl">Page not found</h1>
+		<p class="govuk-body">
+			If you typed the web address, check it is correct.
+		</p>
+		<p class="govuk-body">
+			If you pasted the web address, check you copied the entire address.
+		</p>
+		<p class="govuk-body">
+			If the web address is correct or you selected a link or button, <a href="/contact-us" class="govuk-link">contact us</a> to report a fault.
```

Comment:
This differs a little from the service unavailable page, that one uses "text-danger" for increased spacing, not sure why though, @jimwashbrook do you know? should we bring them in line for conistency?

Reply:
The GDS template didn't have `text-danger` - happy to add it though. Perhaps a question for Sam?

Reply:
oh actually we don't even define any styling with that css class, looks like its old, the increased spacing is just because the _lack_ of `govuk-body` means it doesn't have the margin removed

on the flip side, actually I reckon, we should update them all to standard like you've done, id ping it to sam yeah, that could be another PR though

Reply:
`text-danger` is from bootstrap, which comes with the boilerplate app.

Reply:
Sam seems happy with the screenshots I've sent him. Will probs be easier for him to review something deployed to dev though in terms of the actual classes etc